src/vllm_tgis_adapter/tgis_utils/logs.py

```diff
@@ -138,7 +141,8 @@ def _log_response(  # noqa: PLR0913
         f"inference_time={inference_time * 1e3:.2f}ms "
         f"time_per_token={time_per_token * 1e3:.2f}ms "
         f"total_time={total_time * 1e3:.2f}ms "
-        f"input_toks={response.input_token_count}}}"
+        f"input_toks={response.input_token_count} "
+        f"x-correlation-id={headers.get('x-correlation-id')}}}"
```

Comment:
Per Basker's comment, use this name for compatibility with TGIS logs
```suggestion
        f"correlation_id={headers.get('x-correlation-id')}}}"
```